lib/_http_client.js

```diff
@@ -547,7 +547,12 @@ function parserOnIncomingClient(res, shouldKeepAlive) {
       req.emit('continue');
     }
     // Send information events to all 1xx responses except 101 Upgrade.
-    req.emit('information', { statusCode: res.statusCode });
+    req.emit('information', {
+      statusCode: res.statusCode,
+      statusMessage: res.statusMessage,
+      headers: res.headers,
+      rawHeaders: res.rawHeaders
+    });
```

Comment:
Can you also update the documentation for the `'information'` event in `doc/api/http.md`?